contrib/charts/cert-manager/values.yaml

```diff
@@ -64,3 +64,7 @@ ingressShim:
 # namespace manifest for the namespace that cert-manager is being installed
 # within. It should **not** be used if you are using Helm for deployment.
 createNamespaceResource: false
+clusterIssuer:
+  name: letsencrypt
+  server: https://acme-v01.api.letsencrypt.org/directory
```

Comment:
Should probably default to the staging endpoint and link to docs on how to switch instead of putting prod straight away. Angry users otherwise 😠

Reply:
nit: can you move this stanza above to `createNamespaceResource` option (as it's not really an option for users to set so I want to keep it separate)